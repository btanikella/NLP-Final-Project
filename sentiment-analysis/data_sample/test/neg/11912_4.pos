DT NNP NNP VBZ RP IN NNS  IN DT NN JJ NN NN RB IN IN NNP IN IN CD CC NN NNP -LRB- DT JJ CD -RRB-
RB DT NN RB VBZ TO VB PRP$ JJ NN IN DT JJ NNP NN WDT VBZ RB RB VBN IN DT RB JJ NNP NNP
PRP VBZ IN IN NN NNP VBD PDT PRP$ NN IN DT JJ CD NNS  RB VBG DT VBN RP DT NN PRP VBD PRP
VB DT NN CC VB IN DT NNP NNP NNP
