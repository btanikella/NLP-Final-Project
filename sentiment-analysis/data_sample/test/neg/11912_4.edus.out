Sentence #1 (30 tokens):
The Shanghai Cobra starts out like gangbusters, with a rain soaked diner scene straight out of Shack Out on 101 or Gun Crazy (the first one).
[Text=The CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=DT] [Text=Shanghai CharacterOffsetBegin=8 CharacterOffsetEnd=16 PartOfSpeech=NNP] [Text=Cobra CharacterOffsetBegin=17 CharacterOffsetEnd=22 PartOfSpeech=NNP] [Text=starts CharacterOffsetBegin=23 CharacterOffsetEnd=29 PartOfSpeech=VBZ] [Text=out CharacterOffsetBegin=30 CharacterOffsetEnd=33 PartOfSpeech=RP] [Text=like CharacterOffsetBegin=34 CharacterOffsetEnd=38 PartOfSpeech=IN] [Text=gangbusters CharacterOffsetBegin=39 CharacterOffsetEnd=50 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=50 CharacterOffsetEnd=51 PartOfSpeech=,] [Text=with CharacterOffsetBegin=52 CharacterOffsetEnd=56 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=57 CharacterOffsetEnd=58 PartOfSpeech=DT] [Text=rain CharacterOffsetBegin=59 CharacterOffsetEnd=63 PartOfSpeech=NN] [Text=soaked CharacterOffsetBegin=64 CharacterOffsetEnd=70 PartOfSpeech=JJ] [Text=diner CharacterOffsetBegin=71 CharacterOffsetEnd=76 PartOfSpeech=NN] [Text=scene CharacterOffsetBegin=77 CharacterOffsetEnd=82 PartOfSpeech=NN] [Text=straight CharacterOffsetBegin=83 CharacterOffsetEnd=91 PartOfSpeech=RB] [Text=out CharacterOffsetBegin=92 CharacterOffsetEnd=95 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=96 CharacterOffsetEnd=98 PartOfSpeech=IN] [Text=Shack CharacterOffsetBegin=99 CharacterOffsetEnd=104 PartOfSpeech=NNP] [Text=Out CharacterOffsetBegin=105 CharacterOffsetEnd=108 PartOfSpeech=IN] [Text=on CharacterOffsetBegin=109 CharacterOffsetEnd=111 PartOfSpeech=IN] [Text=101 CharacterOffsetBegin=112 CharacterOffsetEnd=115 PartOfSpeech=CD] [Text=or CharacterOffsetBegin=116 CharacterOffsetEnd=118 PartOfSpeech=CC] [Text=Gun CharacterOffsetBegin=119 CharacterOffsetEnd=122 PartOfSpeech=NN] [Text=Crazy CharacterOffsetBegin=123 CharacterOffsetEnd=128 PartOfSpeech=NNP] [Text=-LRB- CharacterOffsetBegin=129 CharacterOffsetEnd=130 PartOfSpeech=-LRB-] [Text=the CharacterOffsetBegin=130 CharacterOffsetEnd=133 PartOfSpeech=DT] [Text=first CharacterOffsetBegin=134 CharacterOffsetEnd=139 PartOfSpeech=JJ] [Text=one CharacterOffsetBegin=140 CharacterOffsetEnd=143 PartOfSpeech=CD] [Text=-RRB- CharacterOffsetBegin=143 CharacterOffsetEnd=144 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=144 CharacterOffsetEnd=145 PartOfSpeech=.] 
root(ROOT-0, starts-4)
det(Cobra-3, The-1)
nn(Cobra-3, Shanghai-2)
nsubj(starts-4, Cobra-3)
prt(starts-4, out-5)
prep(starts-4, like-6)
pobj(like-6, gangbusters-7)
punct(starts-4, ,-8)
prep(starts-4, with-9)
det(scene-14, a-10)
nn(scene-14, rain-11)
amod(scene-14, soaked-12)
nn(scene-14, diner-13)
pobj(with-9, scene-14)
advmod(out-16, straight-15)
advmod(scene-14, out-16)
prep(out-16, of-17)
pobj(of-17, Shack-18)
prep(with-9, Out-19)
pcomp(Out-19, on-20)
pobj(on-20, 101-21)
cc(101-21, or-22)
conj(101-21, Gun-23)
dep(101-21, Crazy-24)
punct(one-28, -LRB--25)
det(one-28, the-26)
amod(one-28, first-27)
appos(Crazy-24, one-28)
punct(one-28, -RRB--29)
punct(starts-4, .-30)

Sentence #2 (27 tokens):
Unfortunately the film then proceeds to plod its weary way through a standard Chan formula that is only barely enlivened by the always wonderful Mantan Moreland.
[Text=Unfortunately CharacterOffsetBegin=150 CharacterOffsetEnd=163 PartOfSpeech=RB] [Text=the CharacterOffsetBegin=164 CharacterOffsetEnd=167 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=168 CharacterOffsetEnd=172 PartOfSpeech=NN] [Text=then CharacterOffsetBegin=173 CharacterOffsetEnd=177 PartOfSpeech=RB] [Text=proceeds CharacterOffsetBegin=178 CharacterOffsetEnd=186 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=187 CharacterOffsetEnd=189 PartOfSpeech=TO] [Text=plod CharacterOffsetBegin=190 CharacterOffsetEnd=194 PartOfSpeech=VB] [Text=its CharacterOffsetBegin=195 CharacterOffsetEnd=198 PartOfSpeech=PRP$] [Text=weary CharacterOffsetBegin=199 CharacterOffsetEnd=204 PartOfSpeech=JJ] [Text=way CharacterOffsetBegin=205 CharacterOffsetEnd=208 PartOfSpeech=NN] [Text=through CharacterOffsetBegin=209 CharacterOffsetEnd=216 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=217 CharacterOffsetEnd=218 PartOfSpeech=DT] [Text=standard CharacterOffsetBegin=219 CharacterOffsetEnd=227 PartOfSpeech=JJ] [Text=Chan CharacterOffsetBegin=228 CharacterOffsetEnd=232 PartOfSpeech=NNP] [Text=formula CharacterOffsetBegin=233 CharacterOffsetEnd=240 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=241 CharacterOffsetEnd=245 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=246 CharacterOffsetEnd=248 PartOfSpeech=VBZ] [Text=only CharacterOffsetBegin=249 CharacterOffsetEnd=253 PartOfSpeech=RB] [Text=barely CharacterOffsetBegin=254 CharacterOffsetEnd=260 PartOfSpeech=RB] [Text=enlivened CharacterOffsetBegin=261 CharacterOffsetEnd=270 PartOfSpeech=VBN] [Text=by CharacterOffsetBegin=271 CharacterOffsetEnd=273 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=274 CharacterOffsetEnd=277 PartOfSpeech=DT] [Text=always CharacterOffsetBegin=278 CharacterOffsetEnd=284 PartOfSpeech=RB] [Text=wonderful CharacterOffsetBegin=285 CharacterOffsetEnd=294 PartOfSpeech=JJ] [Text=Mantan CharacterOffsetBegin=295 CharacterOffsetEnd=301 PartOfSpeech=NNP] [Text=Moreland CharacterOffsetBegin=302 CharacterOffsetEnd=310 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=310 CharacterOffsetEnd=311 PartOfSpeech=.] 
root(ROOT-0, proceeds-5)
advmod(proceeds-5, Unfortunately-1)
det(film-3, the-2)
nsubj(proceeds-5, film-3)
advmod(proceeds-5, then-4)
aux(plod-7, to-6)
xcomp(proceeds-5, plod-7)
poss(way-10, its-8)
amod(way-10, weary-9)
dobj(plod-7, way-10)
prep(plod-7, through-11)
det(formula-15, a-12)
amod(formula-15, standard-13)
nn(formula-15, Chan-14)
pobj(through-11, formula-15)
nsubj(enlivened-20, that-16)
cop(enlivened-20, is-17)
advmod(enlivened-20, only-18)
advmod(enlivened-20, barely-19)
rcmod(formula-15, enlivened-20)
prep(enlivened-20, by-21)
det(Moreland-26, the-22)
advmod(wonderful-24, always-23)
amod(Moreland-26, wonderful-24)
nn(Moreland-26, Mantan-25)
pobj(by-21, Moreland-26)
punct(proceeds-5, .-27)

Sentence #3 (27 tokens):
It's as if director Karlson blew half his budget in the first five minutes, just getting the set up the way he wanted it.
[Text=It CharacterOffsetBegin=316 CharacterOffsetEnd=318 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=318 CharacterOffsetEnd=320 PartOfSpeech=VBZ] [Text=as CharacterOffsetBegin=321 CharacterOffsetEnd=323 PartOfSpeech=IN] [Text=if CharacterOffsetBegin=324 CharacterOffsetEnd=326 PartOfSpeech=IN] [Text=director CharacterOffsetBegin=327 CharacterOffsetEnd=335 PartOfSpeech=NN] [Text=Karlson CharacterOffsetBegin=336 CharacterOffsetEnd=343 PartOfSpeech=NNP] [Text=blew CharacterOffsetBegin=344 CharacterOffsetEnd=348 PartOfSpeech=VBD] [Text=half CharacterOffsetBegin=349 CharacterOffsetEnd=353 PartOfSpeech=PDT] [Text=his CharacterOffsetBegin=354 CharacterOffsetEnd=357 PartOfSpeech=PRP$] [Text=budget CharacterOffsetBegin=358 CharacterOffsetEnd=364 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=365 CharacterOffsetEnd=367 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=368 CharacterOffsetEnd=371 PartOfSpeech=DT] [Text=first CharacterOffsetBegin=372 CharacterOffsetEnd=377 PartOfSpeech=JJ] [Text=five CharacterOffsetBegin=378 CharacterOffsetEnd=382 PartOfSpeech=CD] [Text=minutes CharacterOffsetBegin=383 CharacterOffsetEnd=390 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=390 CharacterOffsetEnd=391 PartOfSpeech=,] [Text=just CharacterOffsetBegin=392 CharacterOffsetEnd=396 PartOfSpeech=RB] [Text=getting CharacterOffsetBegin=397 CharacterOffsetEnd=404 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=405 CharacterOffsetEnd=408 PartOfSpeech=DT] [Text=set CharacterOffsetBegin=409 CharacterOffsetEnd=412 PartOfSpeech=VBN] [Text=up CharacterOffsetBegin=413 CharacterOffsetEnd=415 PartOfSpeech=RP] [Text=the CharacterOffsetBegin=416 CharacterOffsetEnd=419 PartOfSpeech=DT] [Text=way CharacterOffsetBegin=420 CharacterOffsetEnd=423 PartOfSpeech=NN] [Text=he CharacterOffsetBegin=424 CharacterOffsetEnd=426 PartOfSpeech=PRP] [Text=wanted CharacterOffsetBegin=427 CharacterOffsetEnd=433 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=434 CharacterOffsetEnd=436 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=436 CharacterOffsetEnd=437 PartOfSpeech=.] 
root(ROOT-0, 's-2)
nsubj('s-2, It-1)
mark(blew-7, as-3)
mark(blew-7, if-4)
nn(Karlson-6, director-5)
nsubj(blew-7, Karlson-6)
advcl('s-2, blew-7)
predet(budget-10, half-8)
poss(budget-10, his-9)
dobj(blew-7, budget-10)
prep(blew-7, in-11)
det(minutes-15, the-12)
amod(minutes-15, first-13)
num(minutes-15, five-14)
pobj(in-11, minutes-15)
punct('s-2, ,-16)
advmod(getting-18, just-17)
advcl('s-2, getting-18)
dobj(getting-18, the-19)
partmod(the-19, set-20)
prt(set-20, up-21)
det(way-23, the-22)
dobj(set-20, way-23)
nsubj(wanted-25, he-24)
rcmod(way-23, wanted-25)
dobj(wanted-25, it-26)
punct('s-2, .-27)

Sentence #4 (11 tokens):
Watch the beginning and think about The Phenix City Story.
[Text=Watch CharacterOffsetBegin=442 CharacterOffsetEnd=447 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=448 CharacterOffsetEnd=451 PartOfSpeech=DT] [Text=beginning CharacterOffsetBegin=452 CharacterOffsetEnd=461 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=462 CharacterOffsetEnd=465 PartOfSpeech=CC] [Text=think CharacterOffsetBegin=466 CharacterOffsetEnd=471 PartOfSpeech=VB] [Text=about CharacterOffsetBegin=472 CharacterOffsetEnd=477 PartOfSpeech=IN] [Text=The CharacterOffsetBegin=478 CharacterOffsetEnd=481 PartOfSpeech=DT] [Text=Phenix CharacterOffsetBegin=482 CharacterOffsetEnd=488 PartOfSpeech=NNP] [Text=City CharacterOffsetBegin=489 CharacterOffsetEnd=493 PartOfSpeech=NNP] [Text=Story CharacterOffsetBegin=494 CharacterOffsetEnd=499 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=499 CharacterOffsetEnd=500 PartOfSpeech=.] 
root(ROOT-0, Watch-1)
det(beginning-3, the-2)
dobj(Watch-1, beginning-3)
cc(Watch-1, and-4)
conj(Watch-1, think-5)
prep(think-5, about-6)
det(Story-10, The-7)
nn(Story-10, Phenix-8)
nn(Story-10, City-9)
pobj(about-6, Story-10)
punct(Watch-1, .-11)

